data/randomization/randomization-with-weather.ipynb

```diff
@@ -0,0 +1,732 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    dfw = dfw.loc[:, ['Datetime UTC', 'Rain', 'Speed']]
</pre><p>Just nitpicking here: I think you could change "Speed" to "Wind speed" to remove ambiguity when reading this notebook later.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-echospace/bat-detector-validation/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>